superset/config.py

```diff
@@ -1066,6 +1066,12 @@ def SQL_QUERY_MUTATOR(  # pylint: disable=invalid-name,unused-argument
     return sql
 
 
+def NOTIFICATION_EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
+    msg: Dict[str, Any], **kwargs: Any
```

Comment:
One reason we went with `**kwargs` was so that we could keep the information going into the mutator broad and then put the onus of structuring the data on the mutator in `superset_config`. Though thinking about it, this same thought pattern would exist with a `Dict` named parameter. Though in those cases the mutator class would have to account for newer changes, yeah.